seatunnel-engine/seatunnel-engine-server/src/test/resources/batch_fakesource_to_file_with_checkpoint.conf

```diff
@@ -15,13 +15,12 @@
 # limitations under the License.
 #
 ######
-###### This config file is a demonstration of streaming processing in seatunnel config
+###### This config file is a demonstration of batch processing in seatunnel config
 ######
 
 env {
   parallelism = 1
   job.mode = "BATCH"
-  checkpoint.interval = 1000
```

Comment:
Hi @e-mhui , please revert this. You can find the reason from the file name. `batch ... with checkpoint`.

Reply:
Ok, i revert it.